### ISSUE TITLE
changeFeed options

### ISSUE THREAD

**Issue by thaiat** [neumino/rethinkdbdash#92]
Hi,

the driver does not seem to honor any of the options of the changes function...
(squash and includeStates)
am I doing something wrong ?

**Comment by neumino** [comment]
Could you provide a snippet where the options are not honored?
The code should support them.

**Comment by thaiat** [comment]
this is the code i'm using
```js
var r = require('rethinkdbdash')();
r.db(model.dataSource.connector.database).table(modelName)
                    //.filter(query.filter)
                    .orderBy({
                        index: r.desc('age')
                    })
                    .filter({
                        eyeColor: 'brown'
                    })
                    .limit(40)
                    .changes({
                        squash: 4.0
                    })
                    .run({
                        cursor: true
                    }, handleChange);

 var handleChange = function(err, cursor) {
                    if (err) {
                        console.log(err);
                    } else {
                        if (cursor) {
                            cursor.each(function(err, record) {
                                if (err) {
                                    console.log(err);
                                } else {
                                    console.log('CHANGES', record);
                                    socket.emit(modelName + ':changes', record);
                                }
                            });
                        }
                    }
                    socket.on(modelName + ':changes:stop', stopCursor);
                    socket.on('disconnect', stopCursor);

                    function stopCursor() {
                        if (cursor) {
                            cursor.close();
                        }
                        socket.removeListener(modelName + ':changes:stop', stopCursor);
                        socket.removeListener('disconnect', stopCursor);
                    }
                };
```

i would expect a fewer emitted event with the squash options but it does not seem to have any influence

Let me know what you think

**Comment by thaiat** [comment]
i confirm that this is a bug. When switching back to `rethinkdb` the squash works.
One way to verify this bug is to look in `jobs` table. You will see that the exact query runned by rethinkdb has no option in changes when using rethinkdbdash

minimal code to reproduce 
```js
var r = require('rehtinkdbdash')();
r.db('test').table('Customer')
    .limit(40)
    changes({
        squash: 4.0
     }).run();
```

and the working code
```js
var r = require('rethinkdb');

r.connect().then(function(conn) {
  return r.db('test').table('Customer')
    .limit(40)
    .changes({
       squash: 4.0
    })
    .run(conn);
});
```

**Comment by neumino** [comment]
Thanks for the report, fixed in 2.0.11

**Comment by thaiat** [comment]
thanks for the fix, 2.0.11 does not seem to be published on npm....

**Comment by neumino** [comment]
Sorry, I just tagged it and forgot to publish it. It should be out now
